tests/unit/services/ember-cli-test.js

```diff
@@ -0,0 +1,42 @@
+import { moduleFor, test } from 'ember-qunit';
+import Ember from 'ember';
+
+moduleFor('service:ember-cli', 'Unit | Service | ember cli', {
+  // Specify the other units that are required for this test.
+  needs: ['model:gist','model:gistFile']
+});
+
+// Replace this with your real tests.
+test('compiling a gist works', function(assert) {
+  assert.expect(7);
+  var service = this.subject();
+  assert.ok(service);
+
+  var gist = Ember.Object.create({
```

Comment:
using actual models here overcomplicated things, so I chose to use normal objects.